src/typed-error.ts

```diff
@@ -1,42 +1,43 @@
 /// <reference path='ext.d.ts'/>
+
```

Comment:
BTW honestly I don't see which changes are uglier here. The semicolons are mandatory according to our styleguide (I personally prefer without them, but that's what we've agreed to), and the only change other than that is the omission of the parens for a single-arg fat-arrow function. How is this uglier, or at least significantly uglier?